.gitignore

```diff
@@ -2,6 +2,7 @@
 **/node_modules
 
 packages/contracts/generated
+packages/contracts/lib/forge-std
```

Comment:
we want to include this in the repo - this is a link to the commit used by the submodule